types/redis/index.d.ts

```diff
@@ -1209,10 +1210,10 @@ export interface RedisClient extends Commands<boolean>, EventEmitter {
 
     duplicate(options?: ClientOpts, cb?: Callback<RedisClient>): RedisClient;
 
-    sendCommand(command: string, cb?: Callback<any>): boolean;
-    sendCommand(command: string, args?: any[], cb?: Callback<any>): boolean;
-    send_command(command: string, cb?: Callback<any>): boolean;
-    send_command(command: string, args?: any[], cb?: Callback<any>): boolean;
+    sendCommand(command: string, cb?: Callback<any>): any;
```

Comment:
IIRC `sendCommand(cmd)` falls through (in order) to:
* `client[cmd](...args)`
* `client.internal_send_command(cmd, ...args)`

both of which, last i saw, return bools. the result is meant to come back through the callback i thought